src/handbook/product/verticals.md

```diff
@@ -29,6 +89,6 @@ Some of the companies that are in this market include:
 - UiPath
 - HubSpot
```

Comment:
Hubspot is a CRM?

Reply:
The list is from the existing handbook page (https://flowforge.com/handbook/product/verticals/). @ZJvandeWeg do you know the reason HubSpot was included?

Reply:
Maybe because it has a workflow builder, but let's remove